contribution/lang/fr.json

```diff
@@ -145,35 +145,35 @@
  	 	 	},
  	 	 	"toStation": {
  	 	 	 	"name": {
- 	 	 	 	 	"trans": "Aller vers la station d'Hypertrain",
+ 	 	 	 	 	"trans": "Aller vers la Station Centrale d'Hypertrain",
  	 	 	 	 	"eng": "Go to Hyper Train Central Station"
  	 	 	 	},
  	 	 	 	"tut": {
- 	 	 	 	 	"trans": "Débutez votre aventure vers les récompenses épiques ici",
+ 	 	 	 	 	"trans": "Débutez votre aventure pour des récompenses épiques ici",
  	 	 	 	 	"eng": "Start your adventure to epic loots here"
  	 	 	 	},
  	 	 	 	"description": {
- 	 	 	 	 	"trans": "Aller vers la station d'Hypertrain",
+ 	 	 	 	 	"trans": "Aller vers la Station Centrale d'Hypertrain",
  	 	 	 	 	"eng": "Go to Hyper Train Central Station"
  	 	 	 	}
  	 	 	},
  	 	 	"toJob": {
  	 	 	 	"name": {
- 	 	 	 	 	"trans": "Aller vers la zone commerciale de Shangri-La",
+ 	 	 	 	 	"trans": "Aller vers la Zone Commerciale de Shangri-La",
  	 	 	 	 	"eng": "Go to Shangri-La Commercial Area"
  	 	 	 	},
  	 	 	 	"tut": {
- 	 	 	 	 	"trans": "Vous pouvez effectuer des jobs AFK pour améliorer vos compétences ici",
+ 	 	 	 	 	"trans": "Vous pouvez faire des boulots AFK pour améliorer vos compétences ici",
```

Comment:
"faire des boulots" -> "effectuer des travaux"

Reply:
"jobs" is not translated to "travaux". a "job" is a familiar word for "work". as "work" is translated to "travail" in this context, using a familiar word to translate "job" would be "boulot"